tools/openblas_support.py

```diff
@@ -33,7 +33,7 @@ def get_arch():
         ret = 'arm';
     elif 'aarch64' in os.uname()[-1]:
         ret = 'arm';
-    elif 'x86' in os.uname()[-1]:
+    elif 'x86' in os.uname()[-1] or os.uname()[-1] == 'i686':
```

Comment:
Maybe another `elif` as done for `'arm'`. Do we still need the python2 workaround above?

Reply:
ooops, sorry

Reply:
This is 1.18 only, master is very different.

Reply:
Doesn't matter much, hopefully we will soon forget 1.18. I think this is mainly to test azure wheel builds anyway.